composer.json

```diff
@@ -24,8 +24,8 @@
         "ext-soap": "*",
```

Comment:
We might want to drop support for PHP 5.3. I recommend setting it to `>=7.0` since the newest symfony versions will not work with EOL php versions.

Futhermore, I enabled the travis builds for this project, so we should update the `.travis` file to include more recent php versions.